badge-maker/lib/color.spec.js

```diff
@@ -37,7 +37,7 @@ test(normalizeColor, () => {
   given('  blue ').expect('  blue ')
   given('rgb(100%, 200%, 222%)').expect('rgb(100%, 200%, 222%)')
   given('rgb(122, 200, 222)').expect('rgb(122, 200, 222)')
-  given('rgb(100%, 200, 222)').expect('rgb(100%, 200, 222)')
+  given('rgb(122, 200, 222, 1)').expect('rgb(122, 200, 222, 1)')
```

Comment:
This is now allowed by the [CSS Color Module Level 4 draft](https://www.w3.org/TR/css-color-4/#rgb-functions), which treats `rgba` as a legacy form with the same semantics. Badgeseems to render fine as far as I can tell on my bowser.